src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -2233,6 +2242,30 @@ protected CSharpSyntaxNode GetBindableParentNode(CSharpSyntaxNode node)
             return bindableParent;
         }
 
+        internal override Symbol RemapSymbolIfNecessaryCore(Symbol symbol, int position)
+        {
+            Debug.Assert(symbol is LocalSymbol ||
```

Comment:
>symbol is LocalSymbol [](start = 25, length = 21)

I think we should also handle type parameter symbols.

Reply:
Tracked by [https://github.com/dotnet/roslyn/issues/38922](https://github.com/dotnet/roslyn/issues/38922)

---
In reply to: [329699547](https://github.com/dotnet/roslyn/pull/38769#discussion_r329699547) [](ancestors = 329699547)